spec/command-decoder2.spec.ts

```diff
@@ -1,13 +1,45 @@
 import { MessageBuilder } from './../src/util/message-builder';
 import { CommandDecoder2, CommandNodeBuilder } from './../src/command-tree/command-decoder2';
 describe('command-decoder2', () => {
-  it('should auto register first node', () => {
-    let root = new CommandNodeBuilder()
-      .withHelp(() => new MessageBuilder().message('i r helping'))
-      .toNode();
-    let commands = new CommandDecoder2(root);
-    let result = commands.processMessage({ text: 'hi' });
-    expect(result).toBeUndefined
-    ();
-  })
+
+  it('should find simple match', () => {
+    const root = new CommandNodeBuilder()
+      .withCommand(/a/, (msg) => new MessageBuilder().message('a') )
+      .toNode();
+    const commands = new CommandDecoder2(root);
+    const result = commands.processMessage({ text: 'b' });
+    expect(result).toBeFalsy();
```

Comment:
Make this assertion stronger. `toBeFalsy` should be used when you _don't want_ to test if the answer is `0`, `false`, or `''`. Hopefully we know _exactly_ what to expect here.